### PR TITLE
Fix/ws multiple tabs error

### LINKED ISSUE #108: Bug in Spaces of websocket
When i use one tab of muzer and switch between spaces in one tab by going back and select different space it works fine . But when i open lets say two tabs of spaces from same account . Then it does not properly update the data of specific spaces . Spaces data merging with each other.

### PR DESCRIPTION
**Pull request by Vishawdeep-Singh**
Fixes #108

### CODE REVIEW

**Comment by Vishawdeep-Singh** [comment]
@SujithThirumalaisamy Hey , Can you merge this too . I encountered a bug #108 regarding websocket . Fixed it

File: next-app/components/HomeView.tsx [hunk 1]
```diff
@@ -140,9 +140,10 @@ export default function HomeView() {
                     <div className=" text-4xl font-bold">{space.name}</div>
                   </CardHeader>
                   <CardContent className="overflow-visible flex flex-col justify-center  items-center  py-2">
-                    <Image
```
Comment (SujithThirumalaisamy) [inline next-app/components/HomeView.tsx:143]: Why this has been changed? Change this to the `next/image`

Comment (SujithThirumalaisamy) [reply next-app/components/HomeView.tsx:143]: Revert all the `img` to `next/image`

Comment (Vishawdeep-Singh) [reply next-app/components/HomeView.tsx:143]: when i run it , it gives error hostname    "images.unsplash.com" is not configured under images in your `next.config.js`See more info: https://nextjs.org/docs/messages/next-image-unconfigured-host. 

So i thought maybe for now i change it back to img . It was changed to next/Image during linting.

Comment (Vishawdeep-Singh) [reply next-app/components/HomeView.tsx:143]: Same for youtube thumbnail images it gives error hostname is not configured

Comment (Vishawdeep-Singh) [reply next-app/components/HomeView.tsx:143]: @SujithThirumalaisamy  should i still change it ?

Comment (SujithThirumalaisamy) [reply next-app/components/HomeView.tsx:143]: Yeah just add both of them to the domain. 
1. Unsplash
2. Youtube thumb

**Comment by Vishawdeep-Singh** [comment]
@SujithThirumalaisamy

File: next-app/components/OldStreamView.tsx [hunk 2]
```diff
@@ -388,7 +389,7 @@ export default function StreamView({
                     className="bg-gray-800 border-gray-700 shadow-lg hover:shadow-xl transition-shadow"
                   >
                     <CardContent className="p-4 flex flex-col md:flex-row md:space-x-3">
-                      <Image
+                      <img
```
[the remark this answers is not in the document]
Comment (Vishawdeep-Singh) [reply next-app/components/OldStreamView.tsx:392]: pardon me I forgot in oldStreamView

[the remark this answers is not in the document]
Comment (Vishawdeep-Singh) [reply next-app/components/OldStreamView.tsx]: @SujithThirumalaisamy Done sir

**Comment by SujithThirumalaisamy** [comment]
can this PR be made on the main branch? We have the latest spaces changes there. Dont want to remerge all the 11 commits

**Pull request merged**